spec/services/zendesk_sms_service_spec.rb

```diff
@@ -108,7 +108,7 @@
           body here
         BODY
 
-        ["1", "2", "3", "4"].map do |ticket_id|
+        [1, 2, 3, 4].map do |ticket_id|
```

Comment:
nitpick: here and on line 185, prefer `each` to `map`, as `map` implies a transformation in order to return a new array, whereas `each` returns the original enumerable. it doesn't really matter, but 'each' makes clear that no transform is happening, just iteration.